sandbox/src/mocks/r4/searchServiceRequest/responses/ResponseExampleAdvice.json

```diff
@@ -46,16 +40,14 @@
                         ]
                     }
                 ],
-                "code": [
-                    {
-                        "coding": [
-                            {
-                                "system": "https://fhir.nhs.uk/CodeSystem/NHSDigital-servicerequest-code",
-                                "code": "advice-and-guidance"
-                            }
-                        ]
-                    }
-                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "https://fhir.nhs.uk/CodeSystem/servicerequest-code",
```

Comment:
Have we had confirmation that this is the code system? i.e. without the prefix?

Reply:
We should confirm at meeting with IOPS, as this was given to us under the advice endpoints' context.

Reply:
I guess following from the IOPS catch-up we had yesterday this code system could change in the future.